### ISSUE TITLE
BUG - bed verification unhandled error (low priority)

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#416]
Heron pipeline: bed verification step 'Bravo LHR RT => LHR PCR 1 and 2'
If you put a suffixed barcode into Bed 9 (as opposed to beds 4 and 6), you get an unhandled error rather than a nice message.

**To Reproduce**
Steps to reproduce the behaviour:
1. Sequencescape - UAT tab - Test submission - Submission template = 'Limber-Htp - Heron - ...', Library Type = 'PCR amplicon ligated adapters' - copy resulting plate barcode
2. Limber - search using barcode
3. On LHR RT plate screen, click button 'Bravo LHR RT => LHR PCR 1 and 2'
4. On bed verification screen, paste into 'Scan bed' the bed 9 barcode ('580000009659')
5. Paste into 'Scan plate' the RT plate barcode (browser back button in new tab, find barcode on plate screen), and add on the end '-PP1' suffix
6. See error 'The beds could not be validated. There may be network issues, or problems with Sequencescape.'
7. See error in Limber server 'NoMethodError (undefined method `plate_barcode_suffix=' for #<Robots::Bed::Base:0x007fb9d4dd5758>)'
app/models/robots/heron_robot.rb:17:in `block in bed_plates='
app/models/robots/heron_robot.rb:10:in `each'
app/models/robots/heron_robot.rb:10:in `bed_plates='
app/models/robots/robot.rb:50:in `verify'
app/controllers/robots_controller.rb:58:in `verify'

**Expected behaviour**
Get a nice message saying something about how the plate barcode in bed 9 was not as expected.

**Comment by andrewsparkes** [comment]
This bug is obsolete once issue 375 is done (WIP), so put on hold.

**Comment by JamesGlover** [comment]
Should we close this now 375 is in UAT, or wait until its deployed?

**Comment by andrewsparkes** [comment]
When deployed I think.
